tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -520,8 +522,10 @@ func (cfg *IptablesConfigurator) run() {
 		cfg.iptables.InsertRule(iptableslog.UndefinedCommand, constants.ISTIOINBOUND, constants.MANGLE, 1,
 			"-p", constants.TCP, "-m", "mark", "--mark", cfg.cfg.InboundTProxyMark, "-j", constants.RETURN)
 		// prevent intercept traffic from envoy/pilot-agent ==> app by 127.0.0.6 --> podip
-		cfg.iptables.InsertRule(iptableslog.UndefinedCommand, constants.ISTIOINBOUND, constants.MANGLE, 2,
+		cfg.iptables.InsertRuleV4(iptableslog.UndefinedCommand, constants.ISTIOINBOUND, constants.MANGLE, 2,
 			"-p", constants.TCP, "-s", "127.0.0.6/32", "-i", "lo", "-j", constants.RETURN)
+		cfg.iptables.InsertRuleV6(iptableslog.UndefinedCommand, constants.ISTIOINBOUND, constants.MANGLE, 2,
+			"-p", constants.TCP, "-s", "::1/128", "-i", "lo", "-j", constants.RETURN)
```

Comment:
Why not ::6/128 here? Have we get rid of magic 6-suffix ip in ipv6?

Reply:
thats a bug... good catch

Reply:
we have no tests for this right now but I am working on them btw